manufacturing-ai-suite/wind-turbine-anomaly-detection/docs/user-guide/get-started.md

```diff
@@ -2,13 +2,11 @@
 
 -   **Time to Complete:** 30 minutes
 -   **Programming Language:**  Python 3
+<Do we need this??>>
```

Comment:
This was the template we had which every sample app is following so far.
I don't see a harm in keeping or removing this.